### PR TITLE
Configure probot/no-response to allow 28 days when requesting more info on an issue

### PR DESCRIPTION
**Pull request by jasonrudolph**
### Description of the Change

Building on the [probot/no-response](https://github.com/apps/no-response) configuration implemented in #18077, this pull request adjusts the timeframe to allow 28 days for the issue author to reply to a request for more information.

Currently, when someone opens an issue and we need additional information, a maintainer will ask the issue author for more information and the maintainer will apply the [`more-information-needed` label](https://github.com/atom/atom/issues?page=2&q=is%3Aissue+is%3Aopen+label%3Amore-information-needed). Up to six months can go by without the issue author responding to the request for more information. In these cases, because we still lack the requested information, the issue is not actionable. To help Atom's maintainers focus our attention on actionable issues, the [No Response](https://github.com/apps/no-response) app will now close these non-actionable issues after 28 days, and it will post a comment explaining why the issue was closed:

> This issue has been automatically closed because there has been no response to our request for more information from the original author. With only the information that is currently in the issue, we don't have enough information to take action. Please reach out if you have or find the answers we need so that we can investigate further.

At that point, if the original author (or anyone else) replies with the requested information, we can re-open the issue.

### Alternate Designs

N/A

### Possible Drawbacks

When we merge this change, the app's first "sweep" will identify about 40 issues that are more than 28 days old and have the `more-information-needed` label. For anyone watching the atom/atom repository, or anyone subscribed to all of those issues, they'll receive a batch of notifications for all of those issues at once. 📨

### Verification Process

N/A

### CODE REVIEW

**Comment by Arcanemagus** [comment]
Since the bulk of the issues have already been closed by the 180 timeout we probably aren't going to hit any problems, but do we want to try to figure out https://github.com/probot/no-response/issues/24 before  merging this?

**Comment by jasonrudolph** [comment]
> Since the bulk of the issues have already been closed by the 180 timeout we probably aren't going to hit any problems, but do we want to try to figure out [probot/no-response#24](https://github.com/probot/no-response/issues/24) before merging this?

I would indeed like to look into that issue. I'll bring it up during today's sprint planning meeting to see if we can pull it into this sprint.

**Comment by jasonrudolph** [comment]
> Since the bulk of the issues have already been closed by the 180 timeout we probably aren't going to hit any problems, but do we want to try to figure out probot/no-response#24 before merging this?

@Arcanemagus: The team discussed this during the today's sprint planning meeting. We do want to address probot/no-response#24, and we're hoping to do so during this sprint.

Today, when the bug in probot/no-response#24 occurs, the issue author sees that **no-response bot** closed the issue. If the issue author (or anyone else) clicks on **no-response bot**, they can likely understand why the issue was closed:

demo

The issue author is also still free to leave a comment on the issue.

With that in mind, we'd prefer not hold up this change. Unless you feel strongly that this PR should be blocked until probot/no-response#24 is resolved, I'd like to merge this PR and then circle back around to investigate probot/no-response#24. What do you think?

**Comment by Arcanemagus** [comment]
Oh like I said initially we are unlikely to hit it again as we already did the bulk of the closing with the earlier 180 timeout on a response. Fine to merge by me!

**Review by Arcanemagus — APPROVED** [review]

**Pull request merged**
